resources/public/api/conf/2.0/broughtForwardLosses_create.raml

```diff
@@ -9,15 +9,15 @@ is:
   - errors.ruleTaxYearRangeInvalid
   - errors.ruleTaxYearNotEnded
   - errors.formatLossType
-  - errors.formatSelfEmploymentId
-  - errors.ruleSelfEmploymentId
+  - errors.formatBusinessId
+  - errors.ruleBusinessId
   - errors.formatLossAmount
   - errors.ruleLossAmount
   - errors.ruleDuplicateSubmission
   - errors.clientOrAgentNotAuthorised
   - errors.notFoundIncomeSource
```

Comment:
This is the wrong notFound error now - wording was changed so need to use the normal not found wording.